### DISCUSSION TITLE
Discussions do not show up in the mobile notifications section

### DISCUSSION THREAD

**Discussion by retronbv** [community/community#2846] [Mobile] [answered]
I don’t know why but it shows I have a notif, but going into the app it shows I don’t. Upon further inspection this happens with the new beta discussions

**Comment by HPaulson** [comment]
Hey there @YOYITsM3M8! This is actually caused by your watch settings. Go to the repository in which you'd like to receive notifications, then click "Watch". Using the "Participating" option will send you a notification for any discussions you've participated in, or you can choose your own options under "Custom"- Just make sure you have "Discussions" checked. Additionally, make sure that Web & Mobile notifications are enabled in [Settings](https://github.com/settings/notifications).

Doing this will send you notifications for discussions, the same way you receive notifications for PRs and Issues.

Hope this helps! :+1:

**Reply by HPaulson** [reply]
This answer assumes that you're on the GitHub Mobile beta ([More Info](https://github.com/github/feedback/discussions/2846#discussioncomment-171735)); Sorry for not making that clear initially! 
(Honestly forgot that I use testflight)

**Comment by becca** [comment, marked as answer]
The team just pushed an update to GitHub for mobile on iOS and Android as a beta. We'll be supporting Discussions in mobile for everyone shortly so please keep an eye on [the changelog](https://github.blog/changelog/)

**Reply by ghost** [reply]
What do you mean as a beta? Is it just internal testing, or something like a TestFlight? I know about the Feature Preview section in Settings, but there is no discussions option.

**Reply by eliperkins** [reply]
@BenSova You can join the betas for GitHub for iOS and GitHub for Android via [TestFlight](https://testflight.apple.com/join/NLskzwi5) or the [Play Store beta group](https://play.google.com/apps/testing/com.github.android) respectively. 🎊

**Reply by ghost** [reply]
@eliperkins Thanks for the link! I'm really excited to use discussions on mobile, it's a great new feature.

**Reply by retronbv** [reply]
Thank you!
